client/js/components/procedure/basicSettings/ParticipationPhases.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <div class="flex gap-2">
+      <div class="w-8-12">
+        <dp-select
+          :label="{
+            text: labelText,
+            tooltip: helpText
+          }"
+          :name="fieldName"
+          :options="phaseOptions"
+          required
+          v-model="selectedPhase" />
+      </div>
+
+      <div
+        v-if="true || hasPermission('field_phase_iterator')"
+        class="w-4/12 inline-block relative">
+        <dp-label
+          for="r_public_participation_phase_iteration"
+          :text="iterator.label"
+          :tooltip="iterator.tooltip"
+          required />
+        <dp-input
+          name="r_public_participation_phase_iteration"
+          :value="iterator.value"
+          pattern="^[1-9][0-9]*$" />
+      </div>
+    </div>
+
+    <dp-inline-notification
+      :message="permissionMessageText"
+      type="warning" />
+
+    <div
+      v-if="hasPermission('feature_auto_switch_to_procedure_end_phase') && !hasPermission('feature_auto_switch_procedure_phase') && isInParticipation"
+      class="lbl__hint u-mt-0_25 u-mb-0">
+      {{ autoswitchHint }}
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  DpInlineNotification,
+  DpInput,
+  DpLabel,
+  DpSelect
+} from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'ParticipationPhases',
+
+  components: {
+    DpInlineNotification,
+    DpInput,
+    DpLabel,
+    DpSelect
+  },
+
+  props: {
+    autoswitchHint: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    labelText: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    fieldName: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    helpText: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    initSelectedPhase: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    iterator: {
```

Comment:
Why did you call this prop `iterator`? (I can't really imagine what this is, but I see that iterator is used in other places, too. Maybe a doc block would help)
I think we should always add a validator or a doc block to explain the properties an object prop should have, so it's easy to see which data we need to pass into a component

Reply:
Now as You question: I could "hardcode" everything but the iterator.value. Would You prefer that?
Otherwise, I would follow Your suggestion and add a doc-Block and a validator.

Reply:
If it is unlikely that we want to pass in other values for these properties, I think we should "hard code" them. But a short explanation for the `iterator.value` might still be useful. :)

Reply:
I missed, that we still need the unique name. Here we go 9808008566ea3c69576b2dd86890ce78d9b8714b